tests/blockchain/eth/contracts/main/staking_escrow/test_staking_escrow_additional.py

```diff
@@ -843,7 +843,19 @@ def test_wind_down(testerchain, token, escrow_contract, token_economics):
     # Staker deposits some tokens and confirms activity
     sub_stake = token_economics.minimum_allowed_locked
     duration = 10
-    tx = token.functions.transfer(staker, sub_stake).transact({'from': creator})
+
+    def check_last_period():
+        assert sub_stake == escrow.functions.getLockedTokens(staker, duration).call(), "Sub-stake is already unlocked"
+        assert 0 == escrow.functions.getLockedTokens(staker, duration + 1).call(), "Sub-stake is still locked"
+
+    def check_events(value: bool, length: int):
```

Comment:
Perhaps we can rename `value` to `wind_down` to be more explicit?

Reply:
✔️